pom.xml

```diff
@@ -5,18 +5,18 @@
   <description>Activiti Cloud (Parent)</description>
   <groupId>org.activiti.cloud</groupId>
   <artifactId>activiti-cloud-parent</artifactId>
-  <version>7-201711-EA-SNAPSHOT</version>
+  <version>7-201711-EA-salaboy_1560_parent_deps_link-SNAPSHOT</version>
 
   <packaging>pom</packaging>
 
   <properties>
     <version.activiti>7-201711-EA-SNAPSHOT</version.activiti>
     <version.activiti.cloud>${project.version}</version.activiti.cloud>
-    <version.activiti.cloud.service-common>${project.version}</version.activiti.cloud.service-common>
-    <version.activiti.cloud.runtime-bundle>${project.version}</version.activiti.cloud.runtime-bundle>
-    <version.activiti.cloud.audit>${project.version}</version.activiti.cloud.audit>
-    <version.activiti.cloud.query>${project.version}</version.activiti.cloud.query>
-    <version.activiti.cloud.connectors>${project.version}</version.activiti.cloud.connectors>
+    <version.activiti.cloud.service-common>${version.activiti.cloud}</version.activiti.cloud.service-common>
+    <version.activiti.cloud.runtime-bundle>${version.activiti.cloud}</version.activiti.cloud.runtime-bundle>
+    <version.activiti.cloud.audit>${version.activiti.cloud}</version.activiti.cloud.audit>
+    <version.activiti.cloud.query>${version.activiti.cloud}</version.activiti.cloud.query>
+    <version.activiti.cloud.connectors>${version.activiti.cloud}</version.activiti.cloud.connectors>
```

Comment:
Setting the property `version.activiti.cloud` should be done with caution as it will change the dependencies inside the repository itself. For instance, if you set `version.activiti.cloud` inside `runtime-bundle-service` without overriding the property `version.activiti.cloud.runtime-bundle` to `${project.version}` the internal modules will not relying on themselves anymore, but in`version.activiti.cloud`. This can be error prone. Should we override the "self" property on the repositories prior to merge this PR?

Reply:
@erdemedeiros not sure what you mean. 
By doing this, you can control which sub repos are in different feature branches, by just changing this parent pom. 
For example, now I'm working in a feature branch for the connectors repo, and the only thing that I need is to change the version.activiti.cloud.connectors. 
The problem that I was experiencing without this change is that connectors that depends on services-api was trying to resolve services-api to ${project.version} which doesn't exist, because I have only a feature branch of connectors. 
Does this make sense?

Reply:
Are you concerned of a specific situation because of this change?

Reply:
I think this makes a lot of sense. 
Do you want to go and add those properties in each parent pom for each repo? 
I guess that we need to give this a try.